packages/data/addon/mirage/routes/bard-lite.js

```diff
@@ -188,10 +187,12 @@ export default function(
 
         return newRows.concat(
           dimensionValues.map(value =>
-            // TODO figure out why Object.assign refuses to work in Phantom even with Babel polyfill
-            assign({}, currentRow, {
-              [`${dimension}|id`]: value.id,
-              [`${dimension}|desc`]: value.description
+            Object.keys(value).map(key => {
```

Comment:
We're using map but not returning anything in the map callback. Is that the functionality we want?

Reply:
In addition to this, it should map each  key to an object then reduce to one object and use assign. Or just replace the map with forEach

Reply:
ummm.. ya.. my bad.. fixed now.